demo/java/src/main/java/io/github/cosinekitty/astronomy/demo/Main.java

```diff
@@ -1,31 +1,28 @@
 package io.github.cosinekitty.astronomy.demo;
 
 import java.util.Date;
+
 import io.github.cosinekitty.astronomy.AstroTime;
 
 public class Main {
-    private static String UsageText = String.join(System.getProperty("line.separator"),
-        "Command line arguments:",
-        "",
-        "    now",
-        "    Display current date and time using AstroTime.",
-        ""
-    );
+    private static final String usageText = """
```

Comment:
Yes, I started to do this, and I found it was going to be a hassle because it is so new. I don't feel like we need to worry about the latest and greatest version of everything. It's more important to me that Astronomy Engine can be used in the most platforms possible. It's just a library for calculating numbers, so there is no need to be fancy.